__mocks__/required_rules/rule2.json

```diff
@@ -1,6 +1,6 @@
 {
   "customMessage": "Thanks for contributing, everything looks good in your PR description, please give us some time to review.",
   "action": "comment",
-  "eventJsonPath": "$..[?(@.body.match('Issue: '))]",
+  "eventJsonPath": "$..[?(@.body.match(/Issue Reference.*: #[0-9]{4}/) && @.user.login != 'renovate-bot')]",
```

Comment:
Similar comment as earlier with the Issue reference quantifier being `{4}`, but this is a mock and should be fine